api/bin/spark.js

```diff
@@ -13,9 +14,12 @@ const {
   HOST = '127.0.0.1',
   DOMAIN = 'localhost',
   DATABASE_URL,
+  DEAL_INGESTER_TOKEN,
   REQUEST_LOGGING = 'true'
 } = process.env
 
+assert(DEAL_INGESTER_TOKEN, 'DEAL_INGESTER_TOKEN is required')
```

Comment:
TODO: generate a random token; set it in fly secrets for spark-api and deal-observer backend.

Reply:
Done.

@NikolasHaimerl in the deal-observer backend, the service running on Fly.io can find the access token in the environment variable `DEAL_INGESTER_TOKEN`. (Configured via `fly secrets set`.) We will need to send this token in the Authorization request header when submitting new deals from deals-observer to spark-api.

Reply:
@bajtos Could you please also add a comment above the `DEAL_INGESTER_TOKEN` potentially linking to the comment above describing that token should be set on the `deal-observer` side as well 🙏🏻

Reply:
> @bajtos Could you please also add a comment above the `DEAL_INGESTER_TOKEN` potentially linking to the comment above describing that token should be set on the `deal-observer` side as well 🙏🏻

See [cfa2dbd](https://github.com/filecoin-station/spark-api/pull/502/commits/cfa2dbd10d90b6b6f57a6504c6ec074725bba85a)